From: Ben Hutchings <ben@decadent.org.uk>
Date: Thu, 14 Jul 2016 00:30:57 +0100
Subject: bridge: Fix ABI change in 4.6.4
Forwarded: not-needed

Although this structure is private, it affects the symbol version for
br_deliver() which is called from a netfilter module

Hide the new bitfield from genksyms.

---
--- a/net/bridge/br_private.h
+++ b/net/bridge/br_private.h
@@ -304,7 +304,10 @@ struct net_bridge
 	u8				multicast_disabled:1;
 	u8				multicast_querier:1;
 	u8				multicast_query_use_ifaddr:1;
+#ifndef __GENKSYMS__
 	u8				has_ipv6_addr:1;
+	/* 4 bits padding */
+#endif
 
 	u32				hash_elasticity;
 	u32				hash_max;
